source/code/plugin/in_containerinventory.rb

```diff
@@ -190,29 +193,263 @@ def inspectContainer(id, nameMap, clusterCollectEnvironmentVar)
       return containerInstance
     end
 
+    def getContainerInventoryRecords(batchTime, clusterCollectEnvironmentVar)
```

Comment:
This is the same as we do for windows nodes (derive containerinventory from /pods), except that the /pods call is to the kubelet (rather than API server). I assume everything should be the same (except env var part) ??

@rashmichandrashekar - Can you please take a look at this method to check what it is different and if we need to duplicate the parsing code ?

Reply:
@ganga1980 - As we discussed earlier i thought the plan was to refactor the code so that we used kube api parsing to be consistent across win container inventory and these changes?

Reply:
yes, I have a task to refactor this code on the board. let me see if i can get it tomorrow else handle this separate task.

Reply:
done refactoring.